src/python/pants/backend/codegen/protobuf/python/register.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+"""Generate Python sources from Protocol Buffers (Protobufs).
+
+See https://developers.google.com/protocol-buffers/.
+"""
+
+from pants.backend.codegen.protobuf.python import additional_fields
+from pants.backend.codegen.protobuf.python.rules import rules as python_rules
+from pants.backend.codegen.protobuf.target_types import ProtobufLibrary
+
+
```

Comment:
Should this backend (/plugin) go in contrib? We started that conversation a while back, but I don't think that we finished it.

Reply:
It's undecided how much should live in core `pantsbuild.pants` vs. in other repos. 

I think the one thing we mostly agree on is that it should either live in this repository or in another distinct GitHub repository, but not in this repository via `contrib`. Why?

a) An argument for separate repos is that it enforces stability on the Pants API. That doesn't apply if we put it in `contrib` because we could update the call sites like we would normal `src` code.
b) There's more friction for users to activate it. They have to download another wheel.

Maybe we do end up moving this out of `pants.backend` and into a dedicated repo. We still retain that flexibility here through a deprecation.

Reply:
See here for context: https://docs.google.com/document/d/1E6a7nKq5Lf1G-VRS987ZT77KYkR5DT8JNxb9x4I4xEA/edit?usp=sharing

Reply:
I would like to to kill `contrib` as a concept. The idea behind that name was that these are outside contributions, or somehow less blessed than core backends, but that is not how it panned out in practice.

Reply:
We may want to release all official backends as separate plugins, or none of them, but I think their code location should still be pants.backend.